Makefile

```diff
@@ -22,8 +23,12 @@ docker:
 	docker build -t turbo-geth:latest --build-arg git_commit='${GIT_COMMIT}' --build-arg git_branch='${GIT_BRANCH}' --build-arg git_tag='${GIT_TAG}' .
 
 docker-compose:
-	# Uses host's PID,UID,GID. It required to open Erigon's DB from another process (RPCDaemon local-mode)
-	UID_GID=$(shell id -u):$(shell id -g) docker-compose up
+	@if test -n "$(XDG_DATA_HOME)"; then \
```

Comment:
are you sure we need create folders manually? 
On mac folders auto-created under my user (not root).

Reply:
e.g. https://github.com/mariuspod/erigon/blob/devel/docker-compose.yml#L10
The left part of the colon is the local volume path which is expanded and created as root user on linux.

This is causing permission issues later on, that's why I'm pre-creating them with the non-root user on the host so that the volume can just pick it up and not change the permissions at runtime.